components/scripts/gradle/01-validate-incremental-building.sh

```diff
@@ -219,8 +219,8 @@ $(print_separator)
 ${HEADER_COLOR}Measure build results${RESTORE}
 
 Now that the second build has finished successfully, you are ready to measure in
-Gradle Enterprise how well your build leverages Gradle’s local build cache for
-the invoked set of Gradle tasks.
+Gradle Enterprise how well your build leverages Gradle’s incremental building
```

Comment:
```suggestion
Gradle Enterprise how well your build leverages Gradle’s incremental build
```

This is referred to simply as `incremental build` in the docs. I would drop the word `functionality`.

IMO, to be more clear, it would be best to use `up-to-date` instead of `incremental` since this would make it more clear that this differs from incremental task execution.